azure-stack/hci/concepts/azure-stack-hci-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
+description: Learn to view and navigate your Azure Stack HCI dashboard in Azure portal.
+ms.topic: conceptual
+author: alkohli
+ms.subservice: azure-stack-hci
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.date: 02/13/2024
+---
+
+# Azure Stack HCI dashboard
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article details how to view and navigate your Azure Stack HCI dashboard in Azure portal. You can display the dashboard in chart view or in list view.
+
```

Comment:
Change: This article details how to monitor your Azure Stack HCI systems using the dashboards in Azure portal. You can view the status of Azure Stack HCI systems as charts or lists.

Reply:
You also need to add the preview disclaimer include here.

Reply:
Done > updated the description in text and in metadata.

Done > added include: [!INCLUDE [hci-preview](../../includes/hci-preview.md)].